jenkinsfile-runner-steward-image/scripts/run.sh

```diff
@@ -192,6 +194,24 @@ function with_retries() {
   return "$rc"
 }
 
+function check_retry_params() {
+  local rc \
+    cmd=("$@") \
+    retry_interval=$2 \
+    timeout_seconds=$3 \
+
+  if ! [[ "$retry_interval" =~ ^[0-9]+$ ]] || ! [[ "$timeout_seconds" =~ ^[0-9]+$ ]]; then
+    printf """\nOnly integer values are allowed for PIPELINE_CLONE_INTERVAL \
+and PIPELINE_CLONE_TIMEOUT parameters but '%s' and '%s' are passed respectively ...\n""" "$retry_interval" "$timeout_seconds" >&2
```

Comment:
Bash doesn't support triple quotes like Python, but a quoted string can span multiple lines:
```suggestion
    printf "\nOnly integer values are allowed for PIPELINE_CLONE_INTERVAL \
and PIPELINE_CLONE_TIMEOUT parameters but '%s' and '%s' are passed respectively ...\n" "$retry_interval" "$timeout_seconds" >&2
```